lavalink/PlayerManager.py

```diff
@@ -111,6 +112,8 @@ def connected_channel(self):
                 self.queue.insert(0, self.current)
                 await self.play()
                 await self.seek(current_position)
+                if self.node.server_version == 3 and self.node.ws._is_v31:
+                    await self.bulk_set_gain(self.equalizer)

```

Comment:
I believe Lavalink player equalizer settings persist until the filter factory is removed or the player is destroyed (although I trust that if a player is destroyed on the server, Lavalink.py updates to match), although I have no concerns about keeping this line.

Reply:
What does lavaplayer do with the player when it receives a `4006` code from Discord Voice WS?